xml2xls/xls2xml.py

```
@@ -114,7 +114,7 @@ def start_convert(options):
 
     if file_dir is None:
         Log().error("xls files directory can not be empty! try -h for help.")
-        return
+        return aaaaaaa
```

Ruff (0.14.10)

Undefined name `aaaaaaa`

(F821)